src/main/java/sg/ncl/TeamManager2.java

```diff
@@ -2,6 +2,7 @@
 
 import sg.ncl.testbed_interface.Team2;
 
+import java.io.Serializable;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.Serializable'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)